src/User32.Desktop/User32+WindowFromPointFlags.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="WindowFromPointFlags"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// For use with <see cref="ChildWindowFromPointEx"/>
+        /// </summary>
+        [Flags]
+        public enum WindowFromPointFlags
```

Comment:
If it's only used for the one method you document it for, this should be called `ChildWindowFromPointExFlags` per convention.